chimney/src/main/scala-3/io/scalaland/chimney/PartialTransformer.scala

```diff
@@ -168,20 +169,11 @@ object PartialTransformer extends PartialTransformerCompanionPlatform {
     *
     * @since 0.8.0
     */
-  @FunctionalInterface
-  trait AutoDerived[From, To] {
-    def transform(src: From, failFast: Boolean): partial.Result[To]
-  }
-
-  /** @since 0.8.0 */
-  object AutoDerived extends PartialTransformerAutoDerivedCompanionPlatform {
-
-    implicit def liftTotal[From, To](implicit total: Transformer[From, To]): AutoDerived[From, To] =
-      (src: From, failFast: Boolean) => partial.Result.fromCatching(total.transform(src))
-  }
+  type AutoDerived[From, To] = Transformer[From, To]
```

Comment:
Shouldn't this be `PartialTransformer[From, To]`?

Reply:
Well, yes, it should